src/ngraph/frontend/onnx_import/op/pad.cpp

```diff
@@ -73,6 +89,64 @@ namespace ngraph
                 }
 
             } // namespace set_1
+            namespace set_11
+            {
+                NodeVector pad(const Node& node)
+                {
+                    auto data = node.get_ng_inputs().at(0);
+                    auto pads = node.get_ng_inputs().at(1);
+                    std::shared_ptr<ngraph::Node> values;
+                    std::shared_ptr<ngraph::Node> padding_begin;
+                    std::shared_ptr<ngraph::Node> padding_end;
+
+                    if (node.get_ng_inputs().size() == 3)
+                    {
+                        values = node.get_ng_inputs().at(2);
+                    }
+                    else
+                    {
+                        values = ngraph::op::Constant::create(
```

Comment:
We should use `default_opset::Constant` instead of `ngraph::op::Constant` now. Please change all occurences in this file.

Reply:
Or an explicit version; not sure which will turn out to be better in the long run.

Reply:
Done